suite-native/atoms/jest.config.js

```diff
@@ -0,0 +1,4 @@
+const { ...baseConfig } = require('../../jest.config.native');
```

Comment:
We are trying to avoid unecessary boilerplate files for packages as much as possible so I would strongly prefer to just include config in package.json as it is for other packages aka `test:unit: "yarn g:jest -c ../../jest.config.native.js"`

Reply:
If `jest.config.js` is specified, IDE is able to pick it up and use when running test. It is possible to run tests from IDE even without the explicit config, but it is required to define configuration, which is IMO a bit inconvenient.

I can definitely remove it, just sharing my reasoning.

Reply:
Updated.